tests/tracer/test_tracer.py

```diff
@@ -1654,3 +1654,29 @@ def test_fork_manual_span_different_contexts(tracer):
     _, status = os.waitpid(pid, 0)
     exit_code = os.WEXITSTATUS(status)
     assert exit_code == 12
+
+
+def test_fork_different_contexts_span_tags(tracer):
+    root = tracer.trace("root_span")
+    assert root.get_tag("runtime-id") is not None
+    assert root.get_metric(system.PID) is not None
+
+    pid = os.fork()
+
+    if pid:
+        child1 = tracer.trace("span1")
+        assert child1.get_tag("runtime-id") is None
+        assert child1.get_metric(system.PID) is None
```

Comment:
Just to clarify, is this expected behaviour that we don't store `runtime-id` and the process ID on a span in the parent process?

Reply:
correct because the root_span above is from the same process and has these tags